config/manifests/config-manifests-kustomization-control-test.yaml

```diff
@@ -0,0 +1,1219 @@
+apiVersion: v1
```

Comment:
Is this file necessary?

Reply:
we don't need this file as this is an output of kustomize, will delete it, thanks